monai/networks/utils.py

```diff
@@ -606,6 +607,7 @@ def convert_to_onnx(
     rtol: float = 1e-4,
     atol: float = 0.0,
     use_trace: bool = True,
+    do_constant_folding: bool = True,
```

Comment:
Please add the docstring for this new added parameter.

Thank  
Bin

Reply:
Done.